src/components/BookForm.js

```diff
@@ -36,23 +37,32 @@ const BookForm = () => {
 
   return (
```

Comment:
- [ ] The form elements are not aligned properly, kindly consider styling your form so it looks exactly like the zeplin design

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220826_174148](https://user-images.githubusercontent.com/25227022/186952923-96702203-804f-4de6-9d45-c61ac420930c.png)|![Screenshot_20220826_174506](https://user-images.githubusercontent.com/25227022/186953099-f2610031-1f19-49b8-a306-215c944b3959.png)|